src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -1730,6 +1730,50 @@ def test_ps_s3_creation_triggers_on_master_ssl():
         del os.environ['RABBITMQ_CONFIG_FILE']
 
 
+def _get_body(response):
+    body = response['Body']
+    got = body.read()
+    if type(got) is bytes:
+        got = got.decode()
+    return got
+
+
+@attr('sample_test')
```

Comment:
@yuvalif what should be the appropriate attribute used here?

Reply:
it depends which kind of topic you use: HTTP, kafka or amqp